tock/tock/remote_user_auth.py

```diff
@@ -5,11 +5,13 @@
 from django.core.exceptions import ValidationError
 from django.conf import settings
 from django.contrib.auth.models import User
+from django.core.exceptions import PermissionDenied
```

Comment:
Let's combine this with the ValidationError imported on L5 (and while we're at it, we can better match up our imports with best conventions)

Reply:
Whoops, missed that one thanks!